### PR TITLE
chore(performance): categories and rating scales

### PR DESCRIPTION
**Pull request by Carl-Tabuso**
### 🛠 Changes

- Can now perform create and update in performance categories, rating scales, and pre-employment requirements.
- There aren't much here, check commit msgs.

### ✔️ Checklist
This is a set of criteria to ensure the correctness of your PR. Check each that applied.
- [x] Is `staging` the base branch of this PR?
- [x] Do your changes not reveal sensitive information, such as secrets, API keys, etc?
- [x] Are there no erroneous console logs, debuggers, or leftover code in your changes?

### CODE REVIEW

File: resources/views/components/modals/dialog.blade.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@props([
+    'id' => null,
+])
+
+<div class="modal fade" id="{{ $id }}" tabindex="-1" aria-labelledby="{{ $id }}-label" aria-hidden="true">
```
Comment (C-Ivan-Bandilla) [inline resources/views/components/modals/dialog.blade.php:5]: The modal will be out of reference by bootstrap if the modal root container has no wire:ignore.self

**Review by C-Ivan-Bandilla — APPROVED** [review]
Tested 
Add and  update of performance scale

The reordering isn't saving to the database right?

File: resources/views/components/modals/dialog.blade.php (hunk 1 above)
Comment (Carl-Tabuso) [reply resources/views/components/modals/dialog.blade.php:5]: maybe i can use wire:ignore.self modifier even outside the `livewire/` dir. Never considered that.

**Comment by Carl-Tabuso** [comment]
> The reordering isn't saving to the database right?

No, i don't even think that has value at all, it felt unnecessary. I don't think users like to play around the rearrangement.

File: resources/views/components/modals/dialog.blade.php (hunk 1 above)
Comment (Carl-Tabuso) [reply resources/views/components/modals/dialog.blade.php:5]: yup, just missing the modifier. This can be a reusable modal now.

sample
image

**Pull request merged**